viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -189,100 +176,21 @@ CartoFacade::CartoFacade(viam_carto_lib *pVCL, const viam_carto_config c,
     algo_config = ac;
-    path_to_internal_state = config.data_dir + "/internal_state";
     path_to_internal_state_file = config.existing_map;
 };
 
 CartoFacade::~CartoFacade() { bdestroy(config.component_reference); }
 
-void setup_filesystem(std::string data_dir,
-                      std::string path_to_internal_state) {
-    // setup internal state directory if it doesn't exist
-    auto perms =
-        fs::perms::group_all | fs::perms::owner_all | fs::perms::others_read;
-    try {
-        // if directory doesn't exist, create it with expected structure
-        if (!fs::is_directory(data_dir)) {
-            VLOG(1) << "data_dir doesn't exist. Creating " << data_dir;
-            fs::create_directory(data_dir);
-            VLOG(1) << "setting data_dir permissions";
-            fs::permissions(data_dir, perms);
-        }
-        if (!fs::is_directory(path_to_internal_state)) {
-            VLOG(1) << "data_dir's internal_state directory, doesn't exist. "
-                       "Creating "
-                    << path_to_internal_state;
-            fs::create_directory(path_to_internal_state);
-            VLOG(1)
-                << "setting data_dir's internal_state directory's permissions";
-            fs::permissions(path_to_internal_state, perms);
-        }
-    } catch (std::exception &e) {
-        LOG(ERROR) << e.what();
-        throw VIAM_CARTO_DATA_DIR_FILE_SYSTEM_ERROR;
-    }
-}
-
-std::vector<std::string> list_sorted_files_in_directory(std::string directory) {
-    std::vector<std::string> file_paths;
-
-    for (const auto &entry : fs::directory_iterator(directory)) {
-        file_paths.push_back((entry.path()).string());
-    }
-
-    sort(file_paths.begin(), file_paths.end());
-    return file_paths;
-}
-
-std::string get_latest_internal_state_filename(
-    std::string path_to_internal_state) {
-    std::string latest_internal_state_filename;
-
-    std::vector<std::string> internal_state_filename =
-        list_sorted_files_in_directory(path_to_internal_state);
-    bool found_internal_state = false;
-    for (int i = internal_state_filename.size() - 1; i >= 0; i--) {
-        if (internal_state_filename.at(i).find(".pbstream") !=
-            std::string::npos) {
-            latest_internal_state_filename = internal_state_filename.at(i);
-            found_internal_state = true;
-            break;
-        }
-    }
-    if (!found_internal_state) {
-        throw std::runtime_error(
-            "cannot find internal state but they should be present");
-    }
-
-    return latest_internal_state_filename;
-}
-
 void CartoFacade::IOInit() {
     if (state != CartoFacadeState::INITIALIZED) {
         LOG(ERROR) << "carto facade is in state: " << state << " expected "
                    << CartoFacadeState::INITIALIZED;
         throw VIAM_CARTO_NOT_IN_INITIALIZED_STATE;
     }
-    if (config.cloud_story_enabled == true) {
-        slam_mode = determine_slam_mode_cloud_story_enabled(
```

Comment:
Deleted the unused `determine_slam_mode` and renamed `determine_slam_mode_cloud_story_enabled` --> `determine_slam_mode`